src/main/java/Bot.java

```diff
@@ -0,0 +1,162 @@
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class Bot {
+    private final PrintWriter pw = new PrintWriter(System.out);
+    private static final String SEPARATOR = "__________________________________________________________________\n";
+    private static final String INDENTATION = "    ";
```

Comment:
May I suggest using the '\t' control character for indentation to make it more obvious, instead of using spaces?